configcat/README.md

```diff
@@ -0,0 +1,65 @@
+# ConfigCat
+
+![DataDogEvent][3]
+
+## Overview
+
+DataDog integration ensures that all ConfigCat settings changes send to DataDog as an Event. With this feature you can see your system behaviour when changing your settings. You can setup the DataDog integration for a products in the ConfigCat.
```

Comment:
```suggestion
DataDog-ConfigCat  integration ensures that all ConfigCat settings changes are send to DataDog as an Event so that you can see all your system behaviour when changing your settings. 
```